slides/course-04.md

```diff
@@ -226,7 +226,28 @@ interface Addable<A> {
 
 # Typeclasses vs Java interfaces
 
-Since there is no subtyping, TODO @smelc
+Since there is no subtyping (and no concept of _trait object_ or
+_interface object_), we can't build nor work on heterogeneous collections.
+Compare:
+
+[//]: #exdown-skip
+```hs
+-- Can sum a list of @a@s when @a@ has the 'Num' capability.
```

Comment:
```suggestion
-- | Can sum a list of @a@s when @a@ has the 'Num' capability.
```

Let's show the best practices to the students :-)

Reply:
This is what happens when I try to do Haddock without looking at the doc :sweat_smile: